NPE in validating EventBasedGateway

**Describe the bug**
A NullPointerException is thrown on validating a new Deployment, which contains an EventBasedGateway and an intermediate MessageCatchEvent

**To Reproduce**
Model an EventBasedGateway with an intermediate Message Catch Event and leave the Message Definition blank.

**Expected behavior**
A clear and concise description of what went wrong or why this model is invalid.

**Log/Stacktrace**

<details><summary>Full Stacktrace</summary>
 <p>

```
14:02:13.487 [stream-processor] [0.0.0.0:26501-zb-actors-0] ERROR io.zeebe.broker.workflow - Unexpected error while processing resource 'explodingKittens.bpmn'
java.lang.NullPointerException: null
	at java.util.stream.Collectors.lambda$groupingBy$53(Collectors.java:1127) ~[?:?]
	at java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177) ~[?:?]
	at java.util.ArrayList$Itr.forEachRemaining(ArrayList.java:1032) ~[?:?]
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658) ~[?:?]
	at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:274) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[?:?]
	at java.util.ArrayList$Itr.forEachRemaining(ArrayList.java:1032) ~[?:?]
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
	at io.zeebe.model.bpmn.validation.zeebe.EventBasedGatewayValidator.validate(EventBasedGatewayValidator.java:65) ~[zeebe-bpmn-model-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.model.bpmn.validation.zeebe.EventBasedGatewayValidator.validate(EventBasedGatewayValidator.java:34) ~[zeebe-bpmn-model-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.model.bpmn.validation.ValidationVisitor.visit(ValidationVisitor.java:47) ~[zeebe-bpmn-model-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.model.bpmn.traversal.TypeHierarchyVisitor.visit(TypeHierarchyVisitor.java:39) ~[zeebe-bpmn-model-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.model.bpmn.traversal.ModelWalker.walk(ModelWalker.java:60) ~[zeebe-bpmn-model-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.workflow.deployment.transform.BpmnValidator.validate(BpmnValidator.java:35) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.workflow.deployment.transform.DeploymentTransformer.transformResource(DeploymentTransformer.java:100) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.workflow.deployment.transform.DeploymentTransformer.transform(DeploymentTransformer.java:77) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.workflow.deployment.TransformingDeploymentCreateProcessor.processRecord(TransformingDeploymentCreateProcessor.java:59) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.TypedRecordProcessor.processRecord(TypedRecordProcessor.java:46) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.ProcessingStateMachine.lambda$processInTransaction$1(ProcessingStateMachine.java:240) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.db.impl.rocksdb.transaction.ZeebeTransaction.run(ZeebeTransaction.java:69) ~[zeebe-db-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.ProcessingStateMachine.processInTransaction(ProcessingStateMachine.java:231) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.ProcessingStateMachine.processEvent(ProcessingStateMachine.java:200) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.engine.processor.ProcessingStateMachine.readNextEvent(ProcessingStateMachine.java:176) ~[zeebe-workflow-engine-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:128) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:108) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:92) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:196) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
14:02:13.488 [io.zeebe.gateway.impl.broker.BrokerRequestManager] [0.0.0.0:26501-zb-actors-0] ERROR io.zeebe.gateway - Error handling gRPC request
io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Command rejected with code 'CREATE': Expected to deploy new resources, but encountered the following errors:
'explodingKittens.bpmn': null
	at io.grpc.Status.asRuntimeException(Status.java:524) ~[grpc-api-1.22.1.jar:1.22.1]
	at io.zeebe.gateway.EndpointManager.convertThrowable(EndpointManager.java:291) ~[zeebe-gateway-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.gateway.EndpointManager.lambda$sendRequest$3(EndpointManager.java:269) ~[zeebe-gateway-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.gateway.impl.broker.BrokerRequestManager.lambda$sendRequest$1(BrokerRequestManager.java:101) ~[zeebe-gateway-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.gateway.impl.broker.BrokerRequestManager.lambda$sendRequest$3(BrokerRequestManager.java:121) ~[zeebe-gateway-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.gateway.impl.broker.BrokerRequestManager.lambda$sendRequestInternal$5(BrokerRequestManager.java:159) ~[zeebe-gateway-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.future.FutureContinuationRunnable.run(FutureContinuationRunnable.java:27) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorJob.invoke(ActorJob.java:76) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorJob.execute(ActorJob.java:39) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorTask.execute(ActorTask.java:128) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorThread.executeCurrentTask(ActorThread.java:108) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorThread.doWork(ActorThread.java:92) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
	at io.zeebe.util.sched.ActorThread.run(ActorThread.java:196) [zeebe-util-0.21.0-alpha1.jar:0.21.0-alpha1]
Caused by: io.zeebe.gateway.cmd.BrokerRejectionException: Command (CREATE) rejected (INVALID_ARGUMENT): Expected to deploy new resources, but encountered the following errors:
'explodingKittens.bpmn': null
	... 10 more
```

</p>
</details>


**Environment:**
 - OS: [e.g. Linux]
 - Zeebe Version: [e.g. 0.21.0-SNAPSHOT]
 - Configuration: [e.g. exporters etc.]

--- a/bpmn-model/src/main/java/io/zeebe/model/bpmn/util/ModelUtil.java
+++ b/bpmn-model/src/main/java/io/zeebe/model/bpmn/util/ModelUtil.java
@@ -15,17 +15,37 @@
  */
 package io.zeebe.model.bpmn.util;
 
+import static java.util.stream.Collectors.counting;
+import static java.util.stream.Collectors.groupingBy;
+
 import io.zeebe.model.bpmn.instance.Activity;
+import io.zeebe.model.bpmn.instance.Message;
 import io.zeebe.model.bpmn.instance.MessageEventDefinition;
 import java.util.List;
+import java.util.Map.Entry;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 public class ModelUtil {
-  public static List<MessageEventDefinition> getActivityMessageBoundaryEvents(Activity activity) {
+  public static Stream<MessageEventDefinition> getActivityMessageBoundaryEvents(Activity activity) {
+
     return activity.getBoundaryEvents().stream()
         .flatMap(event -> event.getEventDefinitions().stream())
         .filter(definition -> definition instanceof MessageEventDefinition)
-        .map(MessageEventDefinition.class::cast)
+        .map(MessageEventDefinition.class::cast);
+  }
+
+  public static List<String> getDuplicateMessageNames(
+      Stream<MessageEventDefinition> eventDefinitions) {
+
+    final Stream<Message> messages =
+        eventDefinitions
+            .map(MessageEventDefinition::getMessage)
+            .filter(m -> m.getName() != null && !m.getName().isEmpty());
+
+    return messages.collect(groupingBy(Message::getName, counting())).entrySet().stream()
+        .filter(e -> e.getValue() > 1)
+        .map(Entry::getKey)
         .collect(Collectors.toList());
   }
 }

--- a/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ActivityValidator.java
+++ b/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ActivityValidator.java
@@ -18,8 +18,8 @@ package io.zeebe.model.bpmn.validation.zeebe;
 import io.zeebe.model.bpmn.instance.Activity;
 import io.zeebe.model.bpmn.instance.MessageEventDefinition;
 import io.zeebe.model.bpmn.util.ModelUtil;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.List;
+import java.util.stream.Stream;
 import org.camunda.bpm.model.xml.validation.ModelElementValidator;
 import org.camunda.bpm.model.xml.validation.ValidationResultCollector;
 
@@ -31,25 +31,17 @@ public class ActivityValidator implements ModelElementValidator<Activity> {
 
   @Override
   public void validate(Activity element, ValidationResultCollector validationResultCollector) {
-    final Set<String> messageNames = new HashSet<>();
-    ModelUtil.getActivityMessageBoundaryEvents(element)
-        .forEach(
-            event ->
-                validateMessageTriggerUniqueness(validationResultCollector, messageNames, event));
-  }
 
-  private void validateMessageTriggerUniqueness(
-      ValidationResultCollector validationResultCollector,
-      Set<String> boundaryEventMessageNames,
-      MessageEventDefinition messageDefinition) {
-    final String name = messageDefinition.getMessage().getName();
-    final boolean didNotContain = boundaryEventMessageNames.add(name);
+    final Stream<MessageEventDefinition> boundaryEvents =
+        ModelUtil.getActivityMessageBoundaryEvents(element);
+    final List<String> duplicateMessageNames = ModelUtil.getDuplicateMessageNames(boundaryEvents);
 
-    if (!didNotContain) {
-      validationResultCollector.addError(
-          0,
-          String.format(
-              "Cannot have two message catch boundary events with the same name: %s", name));
-    }
+    duplicateMessageNames.forEach(
+        name ->
+            validationResultCollector.addError(
+                0,
+                String.format(
+                    "Multiple message boundary events with the same name '%s' are not allowed.",
+                    name)));
   }
 }

--- a/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/EventBasedGatewayValidator.java
+++ b/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/EventBasedGatewayValidator.java
@@ -19,14 +19,13 @@ import io.zeebe.model.bpmn.instance.EventBasedGateway;
 import io.zeebe.model.bpmn.instance.EventDefinition;
 import io.zeebe.model.bpmn.instance.FlowNode;
 import io.zeebe.model.bpmn.instance.IntermediateCatchEvent;
-import io.zeebe.model.bpmn.instance.Message;
 import io.zeebe.model.bpmn.instance.MessageEventDefinition;
 import io.zeebe.model.bpmn.instance.SequenceFlow;
 import io.zeebe.model.bpmn.instance.TimerEventDefinition;
+import io.zeebe.model.bpmn.util.ModelUtil;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.List;
-import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.camunda.bpm.model.xml.validation.ModelElementValidator;
 import org.camunda.bpm.model.xml.validation.ValidationResultCollector;
@@ -61,13 +60,17 @@ public class EventBasedGatewayValidator implements ModelElementValidator<EventBa
       validationResultCollector.addError(0, ERROR_UNSUPPORTED_TARGET_NODE);
     }
 
-    getMessageEventDefinitions(outgoingSequenceFlows).map(MessageEventDefinition::getMessage)
-        .collect(Collectors.groupingBy(Message::getName, Collectors.counting())).entrySet().stream()
-        .filter(e -> e.getValue() > 1)
-        .forEach(
-            e ->
-                validationResultCollector.addError(
-                    0, "Multiple message catch events with the same name are not allowed."));
+    final Stream<MessageEventDefinition> messages =
+        getMessageEventDefinitions(outgoingSequenceFlows);
+    final List<String> duplicateMessageNames = ModelUtil.getDuplicateMessageNames(messages);
+
+    duplicateMessageNames.forEach(
+        name ->
+            validationResultCollector.addError(
+                0,
+                String.format(
+                    "Multiple message catch events with the same name '%s' are not allowed.",
+                    name)));
 
     if (!succeedingNodesOnlyHaveEventBasedGatewayAsIncomingFlows(element)) {
       validationResultCollector.addError(

--- a/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/MessageValidator.java
+++ b/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/MessageValidator.java
@@ -15,6 +15,7 @@
  */
 package io.zeebe.model.bpmn.validation.zeebe;
 
+import io.zeebe.model.bpmn.instance.BoundaryEvent;
 import io.zeebe.model.bpmn.instance.ExtensionElements;
 import io.zeebe.model.bpmn.instance.IntermediateCatchEvent;
 import io.zeebe.model.bpmn.instance.Message;
@@ -25,6 +26,8 @@ import io.zeebe.model.bpmn.instance.StartEvent;
 import io.zeebe.model.bpmn.instance.zeebe.ZeebeSubscription;
 import java.util.Collection;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.camunda.bpm.model.xml.instance.ModelElementInstance;
 import org.camunda.bpm.model.xml.validation.ModelElementValidator;
 import org.camunda.bpm.model.xml.validation.ValidationResultCollector;
 
@@ -81,11 +84,12 @@ public class MessageValidator implements ModelElementValidator<Message> {
 
   private boolean isReferedByCatchEvent(Message element) {
     final Collection<IntermediateCatchEvent> intermediateCatchEvents =
-        element.getParentElement().getChildElementsByType(Process.class).stream()
-            .flatMap(p -> p.getChildElementsByType(IntermediateCatchEvent.class).stream())
-            .collect(Collectors.toList());
+        getAllElementsByType(element, IntermediateCatchEvent.class);
 
-    return intermediateCatchEvents.stream()
+    final Collection<BoundaryEvent> boundaryEvents =
+        getAllElementsByType(element, BoundaryEvent.class);
+
+    return Stream.concat(intermediateCatchEvents.stream(), boundaryEvents.stream())
         .flatMap(i -> i.getEventDefinitions().stream())
         .anyMatch(
             e ->
@@ -94,11 +98,15 @@ public class MessageValidator implements ModelElementValidator<Message> {
   }
 
   private boolean isReferedByReceiveTask(Message element) {
-    final Collection<ReceiveTask> receiveTasks =
-        element.getParentElement().getChildElementsByType(Process.class).stream()
-            .flatMap(p -> p.getChildElementsByType(ReceiveTask.class).stream())
-            .collect(Collectors.toList());
+    final Collection<ReceiveTask> receiveTasks = getAllElementsByType(element, ReceiveTask.class);
 
     return receiveTasks.stream().anyMatch(r -> r.getMessage() == element);
+  }
+
+  private <T extends ModelElementInstance> Collection<T> getAllElementsByType(
+      Message element, Class<T> type) {
+    return element.getParentElement().getChildElementsByType(Process.class).stream()
+        .flatMap(p -> p.getChildElementsByType(type).stream())
+        .collect(Collectors.toList());
   }
 }

--- a/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ReceiveTaskValidator.java
+++ b/bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/ReceiveTaskValidator.java
@@ -16,8 +16,10 @@
 package io.zeebe.model.bpmn.validation.zeebe;
 
 import io.zeebe.model.bpmn.instance.Message;
+import io.zeebe.model.bpmn.instance.MessageEventDefinition;
 import io.zeebe.model.bpmn.instance.ReceiveTask;
 import io.zeebe.model.bpmn.util.ModelUtil;
+import java.util.stream.Stream;
 import org.camunda.bpm.model.xml.validation.ModelElementValidator;
 import org.camunda.bpm.model.xml.validation.ValidationResultCollector;
 
@@ -35,11 +37,16 @@ public class ReceiveTaskValidator implements ModelElementValidator<ReceiveTask> 
       validationResultCollector.addError(0, "Must reference a message");
     }
 
-    final boolean hasDuplicateMessageListener =
-        ModelUtil.getActivityMessageBoundaryEvents(element).stream()
-            .anyMatch(event -> event.getMessage().getName().equals(message.getName()));
+    final Stream<String> messageNames =
+        ModelUtil.getActivityMessageBoundaryEvents(element)
+            .map(MessageEventDefinition::getMessage)
+            .filter(m -> m.getName() != null && !m.getName().isEmpty())
+            .map(Message::getName);
 
-    if (hasDuplicateMessageListener) {
+    final boolean hasDuplicateMessageName =
+        messageNames.anyMatch(name -> name.equals(message.getName()));
+
+    if (hasDuplicateMessageName) {
       validationResultCollector.addError(
           0, "Cannot reference the same message name as a boundary event");
     }

--- a/bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeEventBasedGatewayValidationTest.java
+++ b/bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeEventBasedGatewayValidationTest.java
@@ -21,6 +21,9 @@ import static java.util.Collections.singletonList;
 import io.zeebe.model.bpmn.Bpmn;
 import io.zeebe.model.bpmn.instance.EventBasedGateway;
 import io.zeebe.model.bpmn.instance.IntermediateCatchEvent;
+import io.zeebe.model.bpmn.instance.Message;
+import io.zeebe.model.bpmn.instance.zeebe.ZeebeSubscription;
+import java.util.Arrays;
 import org.junit.runners.Parameterized.Parameters;
 
 public class ZeebeEventBasedGatewayValidationTest extends AbstractZeebeValidationTest {
@@ -68,7 +71,21 @@ public class ZeebeEventBasedGatewayValidationTest extends AbstractZeebeValidatio
         singletonList(
             expect(
                 EventBasedGateway.class,
-                "Multiple message catch events with the same name are not allowed."))
+                "Multiple message catch events with the same name 'msg' are not allowed."))
+      },
+      {
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .eventBasedGateway()
+            .intermediateCatchEvent(
+                "catch-1", c -> c.message(m -> m.name(null).zeebeCorrelationKey("$.foo")))
+            .moveToLastGateway()
+            .intermediateCatchEvent(
+                "catch-2", c -> c.message(m -> m.name("msg").zeebeCorrelationKey(null)))
+            .done(),
+        Arrays.asList(
+            expect(Message.class, "Name must be present and not empty"),
+            expect(ZeebeSubscription.class, "zeebe:correlationKey must be present and not empty"))
       },
       {
         Bpmn.createExecutableProcess("process")

--- a/bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeMessageValidationTest.java
+++ b/bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeMessageValidationTest.java
@@ -143,6 +143,35 @@ public class ZeebeMessageValidationTest extends AbstractZeebeValidationTest {
         singletonList(
             expect("start-message", "A message cannot be referred by more than one start event"))
       },
+      {
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .serviceTask("task", t -> t.zeebeTaskType("test"))
+            .boundaryEvent(
+                "boundary-1",
+                b -> b.message(m -> m.name(null).zeebeCorrelationKey("correlationKey")))
+            .endEvent()
+            .moveToActivity("task")
+            .boundaryEvent(
+                "boundary-2",
+                b -> b.message(m -> m.name(null).zeebeCorrelationKey("correlationKey")))
+            .endEvent()
+            .done(),
+        Arrays.asList(
+            expect(Message.class, "Name must be present and not empty"),
+            expect(Message.class, "Name must be present and not empty"))
+      },
+      {
+        Bpmn.createExecutableProcess("process")
+            .startEvent()
+            .receiveTask("task")
+            .message(m -> m.name("message").zeebeCorrelationKey("correlationKey"))
+            .boundaryEvent("boundary")
+            .message(m -> m.name(null).zeebeCorrelationKey("correlationKey"))
+            .endEvent()
+            .done(),
+        singletonList(expect(Message.class, "Name must be present and not empty"))
+      },
     };
   }
 

--- a/bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeValidationTest.java
+++ b/bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeValidationTest.java
@@ -96,7 +96,7 @@ public class ZeebeValidationTest extends AbstractZeebeValidationTest {
         singletonList(
             expect(
                 "task",
-                "Cannot have two message catch boundary events with the same name: message"))
+                "Multiple message boundary events with the same name 'message' are not allowed."))
       },
     };
   }

